tensorflow_datasets/testing/tiered_imagenet_generate.py

```diff
@@ -0,0 +1,97 @@
+import pickle
+import os
+import numpy as np
+import tensorflow as tf
+
+_NUM_CATEGORIES_TRAIN = 20
+_NUM_CATEGORIES_VAL = 6
+_NUM_CATEGORIES_TEST = 8
+
+_NUM_CLASSES_TRAIN = 351
+_NUM_CLASSES_VAL = 97
+_NUM_CLASSES_TEST = 160
+_NUM_IMAGES_TRAIN = 44869  # Original size in dataset is 448695

```

Comment:
Ooov! It's a very large number for this testing. And I think you understand wrongly. Yes you can write a generator but then you should run it to fake examples and you should also commit generated fake examples. You should keep them small please.

Reply:
Thanks! I really understood wrong  about the fake examples. I have committed the changes with modified script and generated fake examples.